.github/workflows/ci.yml

```diff
@@ -574,6 +582,7 @@ jobs:
           - mssql
           - mysql
           - postgresql
+          - redshift
```

Comment:
The redshift tests won't actually run until after this PR is merged. We'll see 3 failing tests (these aren't required yet). These are ticketed and I will fix them next.